### PR TITLE
[Doc] Update README.md: a reword suggestion

### PR DESCRIPTION
**Pull request by Sunt-ing**
## Proposed changes
Make the path more readable as the convention of Markdown suggests.

It will not break any existing build.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ If your GCC version is lower than 7.3.0, you can run:
 sudo yum install devtoolset-4-toolchain -y
```
Comment (yangzhg) [inline README.md:131]:
```suggestion
sudo yum install devtoolset-7-toolchain -y
```

File: README.md [hunk 2]
```diff
@@ -131,7 +131,7 @@ If your GCC version is lower than 7.3.0, you can run:
...
 ```
 
-and then, set the path of GCC (e.g /opt/rh/devtoolset-4/root/usr/bin) to the environment variable PATH.
+and then, set the path of GCC (e.g `/opt/rh/devtoolset-4/root/usr/bin`) to the environment variable PATH.
```
Comment (yangzhg) [inline README.md:134]:
```suggestion
and then, set the path of GCC (e.g `/opt/rh/devtoolset-7/root/usr/bin`) to the environment variable PATH.
```

Comment (yangzhg) [reply README.md:134]: you'd better change the Chinese doc too.

**Comment by Sunt-ing** [comment]
@yangzhg I don't find any Chinese version of README.md. Yes, there are at least three readme files: /README.md, /docs/en/README.md, /docs/zh-CN/README.md.
My change here is located in /README.md, which has no paired Chinese version yet.
I globally searched the whole project with the key words "/opt/rh/devtoolset", and only found one match: in /README.md.

**Comment by Sunt-ing** [comment]
By the way, maybe we can mix Chinese and English version docs together in the same directory, and only left the file name different. The benefits are we can find the mismatch of these two docs more easily in this way.
Take Apache Flink as am example: https://github.com/apache/flink/tree/master/docs/internals

**Comment by Sunt-ing** [comment]
@yangzhg I have merged PR#5286 here

**Review by yangzhg — APPROVED** [review]
LGTM

**Pull request merged**